resources/mysqlbinlog/mysqlbinlog_8.0.28-macos11-arm64_test.go

```diff
@@ -0,0 +1,43 @@
+//go:build darwin && arm64
+// +build darwin,arm64
+
+package mysqlbinlog
+
+import (
+	"embed"
+	"os"
+	"os/exec"
+	"path/filepath"
+	"testing"
+
+	"github.com/bytebase/bytebase/resources/utils"
+)
+
+//go:embed mysqlbinlog-8.0.28-macos11-arm64.tar.gz
+var resources embed.FS
+
+// TestBootMySQLBinlog tests whether mysqlbinlog can be started on the target platform
+// to check whether the lib extraction is correct.
+func TestBootMySQLBinlog(t *testing.T) {
```

Comment:
```suggestion
func TestRunBinary(t *testing.T) {
```
IMO "boot" is usually used for a server of computer, and we are only talking about the `mysqlbinlog` binary in this package.
Also, it's hard to pronounce `mysqlbinlog` in the test function name verbatim.

Reply:
SG!